core/src/i18n/en.yaml

```
@@ -1,5 +1,6 @@
 addons:
   addons-status: Addons Status
+  description: <0>Addons Configuration</0> defines Namespace-scoped addons that provide all information required to convert Helm charts into Service Classes in Service Catalog.
```

What about
```suggestion
  description: <0>Addons</0> define Namespace-scoped addons that provide all information required to convert Helm charts into Service Classes in Service Catalog.
```

And the same with Cluster Addons?